File: Javascript/Arrays/Arreglos_sintaxis.js
/*
Arrays en JavaScript, Sintaxis básica de JavaScript para escribir un array.
@Autor: Daniel Alejandro Alvarado Velázquez
@grupo: #2-02
@Fecha: 02/09/2026
*/

//Formas de declarar un arreglo en JavaScript
const arreglo = new Array(5); // Crea un arreglo de tamaño 5
const otro_arreglo = [1, 2, 3, 4, 5]; // Crea un arreglo con valores iniciales
const arreglo_variado = [1, 3.67, "Hola", true]; // una arreglo en javascript puede contener diferentes tipos de datos

// Imprimir los elementos del arreglo
console.log(otro_arreglo[0]); // Imprime 1
console.log(otro_arreglo[4]); // Imprime 5

for (let i = 0; i < arreglo_variado.length; i++) {
    console.log(`El elemento en el indice ${i} es: ${arreglo_variado[i]}`);
}

//Insertamos valores al arreglo vacio
for (let i = 0; i < arreglo.length; i++) {
    arreglo[i] = i + 2;
}
console.log(arreglo);

File: C#/Arrays/Arreglos_sintaxis.cs
/*
Arrays en C#, Sintaxis básica de C# para escribir un array.
@Autor: Daniel Alejandro Alvarado Velázquez
@grupo: #2-02
@Fecha: 02/09/2026
*/

using System;

namespace Arreglos_sintaxis
{
    class Program
    {
        static void Main(string[] args)
        {
            // Sintaxis de un array en C#
            int[] numeros = new int[5];
            int[] arreglo = { 1, 2, 3};
            int[] otro_arreglo = new int[2] { 1, 2 };

            // Asignación de valores al array
            numeros[0] = 10;
            numeros[1] = 20;
            numeros[2] = 30;
            numeros[3] = 40;
            numeros[4] = 50;

            // Impresión de los valores del array en consola
            Console.WriteLine("Valores del array:");
            for (int i = 0; i < numeros.Length; i++)
            {
                Console.WriteLine(numeros[i]);
            }


        }
    }
}